src/content/protocols/aave.md

```diff
@@ -0,0 +1,996 @@
+---
+protocol: "Aave v3"
+website: "https://aave.com"
+x: "https://x.com/aave"
+github: ["https://github.com/aave-dao/aave-v3-origin"]
+defillama_slug: ["aave-v3"]
+chain: "Ethereum"
+stage: 0
+risks: ["L", "H", "H", "H", "L"]
+reasons: []
+author: ["sagaciousyves"]
+submission_date: "1970-01-01"
+publish_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+Aave v3 is a lending protocol that allows users to lend and borrow different ERC20 assets. Users are able to create positions that consist of debt in different loan assets which is secured by different collateral assets. The lending market allows anyone to liquidate insolvent positions, based on an external price feed and specific collateral factors representing an asset's specific risk profile. Furthermore, instead of borrowing supplied assets, Aave V3 also issues its own stablecoin, GHO. Users can borrow and lend GHO like any other asset in the system.
+
+The Aave DAO is Aave's onchain governance system, allowing AAVE, stkAAVE and aAAVE holders to govern over various aspects ranging across treasury management, risk management and strategic initiatives.
+
+Different Aave protocol _Instances_ exist and are managed by the Aave DAO. These instances focus on specific use cases and chains. This review covers the Ethereum Mainnet Core instance.
+
+# Risks
+
+## Chain
+
+The report is concerned with the Aave V3 _Core_ instance deployed on Ethereum mainnet. Ethereum mainnet achieves a _Low_ centralization risk score.
+
+> Chain score: Low
+
+## Upgradeability
+
+The Aave v3 protocol can be analyzed in a number of logical modules: _Core Lend & Borrow_, _Reserve Parameters_, _Treasury, Aave Ecosystem Reserves & Rewards_, _GHO Stablecoin_ and _Aave Governance_. Each module exposes various degrees of control as explained in more detail below. Overall, these control vectors could result in the loss of user funds, loss of unclaimed yield or otherwise materially affect the expected performance of the protocol. All the control vectors, apart from the `EmergencyAdmin`'s priviledge, are behind governance vote or if permissions are given to a multi-sig account, the permission is successfully restricted to prevent mis-use by using steward contracts.
```

Comment:
EmergencyAdmin multisig should be formatted with hyperlink instead of contract annotation